services/common/src/redux/reducers/authenticationReducer.ts

```diff
@@ -76,11 +83,12 @@ const authenticationReducerObject = {
   [AUTHENTICATION]: authenticationReducer,
 };
 
+export const getSystemFlag = (state: RootState) => state[AUTHENTICATION].systemFlag;
 export const isAuthenticated = (state: RootState) => state[AUTHENTICATION].isAuthenticated;
 export const getUserAccessData = (state: RootState) => state[AUTHENTICATION].userAccessData;
 export const getUserInfo = (state: RootState) => state[AUTHENTICATION].userInfo;
 export const userHasRole = (state: RootState, role: string) =>
-  state[AUTHENTICATION].userAccessData.includes(USER_ROLES[role]);
+  state[AUTHENTICATION].userAccessData.includes(USER_ROLES[role] ?? role);
```

Comment:
this was specifically so that I didn't have to bring over _two_ files detailing our roles into common (the Permissions one is completely redundant and obfuscating IMO). But I left the first part of the null coalesce for backwards compatibility.